README.md

```diff
@@ -11,168 +13,170 @@ GodTools
 
-###### ViewModel Responsibilities:
-- Provides data outputs to the View which the View can observe and react to.
-- Each data output connects to a UseCase (Domain Layer) which computes data.  Computing can also happen directly in the ViewModel although UseCases are preferred.
+- Project Fastlane Fastfile (https://github.com/CruGlobal/godtools-swift/blob/develop/fastlane/Fastfile) points to shared Fastfile (https://github.com/CruGlobal/cru-fastlane-files/blob/master/Fastfile)
 
-#### Domain Layer
+- Uses a combination of Fastlane Environment Variables(https://github.com/CruGlobal/godtools-swift/blob/develop/fastlane/.env.default) and GitHub Secrets
 
-The domain layer is responsible for the business rules of the application and is broken up into Use Cases.  The benefits of the domain layer include:
-- It avoids code duplication.
-- It improves readability in ViewModels that use domain layer use cases.
-- It improves testability of the app.
-- It avoids large ViewModels by allowing you to split responsibilities.
+- GitHub Actions Build Workflow: https://github.com/CruGlobal/godtools-swift/blob/develop/.github/workflows/build.yml
 
+- GitHub Actions OneSky Workflow: https://github.com/CruGlobal/godtools-swift/blob/develop/.github/workflows/download_onesky_translations.yml
 
-##### - Use Cases
+- GitHub Actions OneSky Workflow Dependency Plugin: https://github.com/thekie/fastlane-plugin-onesky
 
-Use cases make up the domain layer.  Use cases should be responsible for a single task and named to reflect that task.  For example: AuthenticateUserUseCase, GetSpotlightToolsUseCase.
+#### Conventions
 
-UseCase Best Practices:
+- [Classes](#classes)
+- [Project Folder Structure](#project-folder-structure)
 
-- Should have a single exposed method (public, internal) that takes zero or more inputs and produces a single output that is an AnyPublisher. 
-- UseCases can have private methods, however, as we move to dependency inversion I think private methods will become less and less.
-- Inputs should not be publisher types. Instead the ViewModel should react to changes which then triggers the UseCase.
-- Should not depend on UseCases. If specific Output from a UseCase is needed, instead inject that Output as an Input from the ViewModel.
-- Should depend only on interfaces. Most of the time we depend on some type of Repository Interface where a Repository is simply a data storage and data access.
-- Would prefer that UseCases return a non Swift type and instead some type of DomainModel that encapsulates attributes related to the business requirements.
+##### Classes
 
-Responsibilities:
+- Class / Struct attributes should always be declared with the type.
+- Class level attributes should also be grouped by constant vs instance as well as by access level and mutability. Where constants are at the top and grouped by private first, then internal, then public.  Instance variables follow constants and are grouped by private first, then internal, then public.   
```

Comment:
It might be helpful to have a screenshot of grouped attributes as an example.  It makes sense, but I think it's clearer in a picture than reading (or accidentally misreading) a list of the ordering.

Reply:
Thanks @rachaelblue for the feedback.  I will be updating these areas.

Reply:
Hey @rachaelblue just updated both of those areas.  Please let me know if you have any additional comments or concerns or suggestions. 
For the attribute grouping I don't want to get too nit picky in this area.  Just as long as we have some consistency that is close to what is outlined in the ReadMe.  The main idea is that code is consistent which can be helpful when viewing other people's code in a project.  
I also realize there may be cases for attributes that aren't outlined in the ReadMe, example fileprivate and more special attributes such as Binding, State, etc.  As long as we hit the 3 main groupings static, instance, special, and from there do the best we can to fill in the inner groupings.